draft-ietf-tls-rfc4492bis.xml

```diff
@@ -101,7 +101,8 @@
         generation of bulk encryption/MAC keys and initialization vectors is independent of the key exchange
         algorithm and not impacted by the introduction of ECC.</t>
       <t> <xref target="tbl2"/> summarizes the new key exchange algorithms.  All of these key exchange algorithms
-        provide forward secrecy.</t>
+        provide forward secrecy if and only if fresh ephemeral keys are generated and used, and also destroyed
```

Comment:
Sorry, I  meant the opposite, namely that you are implying that you only get FS if you generate a fresh ephemeral for each connection rather than reusing it a few times. That seems like a debate perhaps we don't want to get dragged into here?